astropy/io/fits/tests/test_hdulist.py

```diff
@@ -913,3 +914,38 @@ def test_proper_error_raised_on_non_fits_file_with_unicode(self):
         # This should raise an IOError because there is no end card.
         with pytest.raises(IOError):
             fits.open(filename)
+
+    # Avoid a spurious pyflakes warning about an unused import
+    # of six by adding a reason string
+    @pytest.mark.skipif(six.PY2, reason='six.PY2')
+    @pytest.mark.parametrize('ignore', [False, True])
```

Comment:
To me it's clearer that it's iterating through both the `ignore` and `fileobj` arguments (i.e. you get 2 * 2 combinations of tests). If I write

    @pytest.mark.parametrize('ignore,fileobj', [(False, False), (False, True), (True, False), (True,True)])

it is harder for me to check that I have all combinations.